README.md

```diff
@@ -125,6 +125,210 @@ record = client.write(record_type, data)
+strawberry_data = {"recipe": "encrypted_secret_formula"}
+strawberry_plain = {"flavor":"strawberry"}
+client.write("jam", strawberry_data, strawberry_plain )
+
+query = Search().Match(record_type=["jam"], values=["apricot"]).Exclude(values=["strawberry"])
+results = client.Search(query)
+```
+
+To filter queries provide a datetime range and a valid timezone, to override the `UTC` default. Run the following search to get all records written by a specific user during the last 24 hours:
+```python
+# e3db setup...
+from datetime import datetime, timedelta
+
+# time T between after <-T-> before
+before = datetime.now()
+after =  before - timedelta(days=1)
+writer_id = "some_writer_uuid"
+
+query = Search().Match(writer=[writer_id]).Range(zone_offset="-08:00", after=after, before=before)
+results = client.Search(query)
+```
+
+#### Defaults
+
+The Search method has a number of default parameters when searching, more detail can be found within the inline documentation.
+
+Under Search there are these defaults:
+```python
+# Optional value that starts search from the first page of results, only required for paginated queries.
+after_index = 0
+
+# Amount of records to be returned, limiting if more are available. Defaults to 50, and the maximum value allowed is 1000.
+count = 50
+
+# Include only records written by the client (False), or search other writer ids (True)
+include_all_writers = False
+
+# Include data when returning records
+include_data = False
+```
+
+Under Search Params there are these defaults:
+```python
+# Conditional OR when searching upon all fields within this Param object
+condition = "OR" # options: "OR"|"AND"
+
+# Exactly matches the search fields provided
+strategy = "EXACT" # options "EXACT"|"FUZZY"|"WILDCARD"|"REGEXP"
+```
+
+Under Search Range there is this default:
+```python
+# Search records based on when they were created or last modified
+key = "CREATED" # options: "CREATED"|"MODIFIED"
+
+# Default time provided is set to UTC.
+zone = "UTC" # options are "PST":"-08:00", "MST":"-07:00", "CST":"-06:00", "EST":"-05:00", "UTC":"+00:00"
+
+# Zone offset is None by default, but if provided it will override the default zone parameter
+zone_offset = None
+```
+
+Since python datetime objects are zone agnostic, provide the proper timezone offset from UTC
+in zone_offset to search properly.
+
+### Advanced Matching Strategies
+
+Search offers advanced queries that provide more flexibility than the default matching strategy of `EXACT`. The four options are `EXACT`, `FUZZY`, `WILDCARD`, and `REGEXP`. Depending on the matching strategy and the fields provided, these searches may be slower. 
+
+To mirror some of the above queries with these matching strategies we get:
+```python
+# e3db setup...
+
+# fuzzy
+# generates an edit distance and matches fields that are 1-2 edits away from the provided query.
+# summer is 1 edit s-> b away from bummer
+fuzz_query = Search().Match(strategy="FUZZY", record_type=["season"], values=["bummer"])
```

Comment:
Just noting the discussion we had regarding multiple match statements being anded vs. orred

Reply:
https://docs.google.com/document/d/1QMZgx-A_o1xkZcJNtCh9x4QkKyE6eF5Yi_V3d8CvmnY/edit#
Page 6
>Search evaluation is equal to the OR of all matches AND’d with the OR of all excludes AND’d with the query range. Put another way, only records that match ANY of the filters in the match clause AND don’t match ANY of the filters in the exclude clause AND that have range key values that intersect with the range clause are returned.